catalog/security/catalog-security-metacardattributeplugin/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -19,25 +19,21 @@
          id="org.codice.ddf.catalog.security.policy.metacard.MetacardAttributeSecurityPolicyPlugin">
 
         <AD name="Metacard Intersect Attributes:" id="intersectMetacardAttributes"
-            description="Metacard attributes that will be collected and mapped to security information. Example: security.classification=classification
-            Any duplicate destinations in this group will result in an intersection of values. E.g. source1=dest and source2=val
-            where source1 is the set of ['a', 'b', 'c'] and source2 is the set ['a', 'c', 'd'] will result in
-            a dest of ['a', 'c']. Note: duplicate destinations for Intersect and Union attributes will
-            behave unpredictably."
+            description="List of metacard attributes whose values will be collected (source attributes) and combined
+            into a (new) metacard attribute (destination attribute). The format is the name of a source attribute,
+            folowed by an equals sign, followed by a destination attribute. Intersection means that only the values
+            common to all source attributes become values of the destination attribute.
+            Note: duplicate destinations for Intersect and Union attributes will behave unpredictably."
             required="true" type="String" cardinality="1000"
             default=""/>
 
         <AD name="Metacard Union Attributes:" id="unionMetacardAttributes"
-            description="Metacard attributes that will be collected and mapped to security information. Example: security.classification=classification
-            Any duplicate destinations in this group will result in a union of values. E.g. source1=dest and source2=val
-            where source1 is the set of ['a', 'b', 'c'] and source2 is the set ['a', 'c', 'd'] will result in
-            a dest of ['a', 'b', 'c', 'd']. Note: duplicate destinations for Intersect and Union attributes will
-            behave unpredictably."
+            description="Similar to Intersect Attributes. The logic of how to combine the source values is different.
```

Comment:
❗️ This is effectively a tooltip that says "see other tooltip". I understand the reasoning, but feel it is a worse user experience. I would recommend fully documenting both attributes, maybe putting the unique section first.
`This is what this attribute does (unique). This is how to configure this attribute (similar).`